### PR TITLE
#9: connect jbrowse to score

### PR DESCRIPTION
**Pull request by samrichca**
DMS-UI PR https://github.com/overture-stack/dms-ui/pull/159

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
-  "presets": ["@babel/preset-env", "@babel/preset-react", "@babel/preset-typescript"],
+  "presets": [
+    "@babel/preset-env",
+    ["@babel/preset-react", { "runtime": "automatic" }],
```
Comment (samrichca) [inline .babelrc:4]: without this I get "react is not defined" error

File: src/components/common.tsx [hunk 2]
```diff
@@ -48,81 +47,13 @@ export const getTracks = (inputFiles: JbrowseFileInput[]) =>
...
         location: {
           uri: input.indexURI,
-          locationType: "UriLocation",
+          locationType: 'UriLocation',
         },
       },
     },
   }));
-
-export const tempFileRoot: string = "http://localhost:3000/data/";
```
Comment (samrichca) [inline src/components/common.tsx:62]: removed all the local/temp setup

File: src/components/common.tsx [hunk 3]
```diff
@@ -17,25 +17,24 @@
...
+export const locationTypes: JbrowseFileFormatDict = {
+  BAM: 'bamLocation',
+  VCF: 'vcfGzLocation',
 };
 
-export const trackTypes: JbrowseFileInputInfo = {
-  BAM: "AlignmentTrack",
-  VCF: "VariantTrack",
+export const trackTypes: JbrowseFileFormatDict = {
+  BAM: 'AlignmentsTrack',
```
Comment (samrichca) [inline src/components/common.tsx:33]: had the wrong API details for BAM files

BAM - Bam
Alignment - Alignments

File: src/components/JbrowseCircular.tsx [hunk 4]
```diff
@@ -28,10 +28,10 @@ import { defaultCircularOptions } from '../utils/circular/dynamic';
 export type CircularViewModel = ReturnType<typeof createViewState>;
 
 export const JbrowseCircular = ({
-  options,
+  configuration,
   selectedFiles = [],
 }: {
-  options?: CircularViewModel;
+  configuration?: CircularViewModel['config']['configuration'];
```
[the remark this answers is not in the document]
Comment (justincorrigible) [reply src/components/JbrowseCircular.tsx:34]: hmm I do remember... I just don't love the nesting of synonyms here.

**Review by justincorrigible — APPROVED** [review]

**Pull request merged**